packages/back-end/src/services/stats.ts

```diff
@@ -251,90 +272,91 @@ export async function analyzeExperimentResults({
     });
   }
 
-  const dimensionMap: Map<string, ExperimentReportResultDimension> = new Map();
-  await promiseAllChunks(
-    metricRows.map((data) => {
+  const results = await analyzeExperimentMetric({
+    coverage: snapshotSettings.coverage ?? 1,
+    phaseLengthHours: Math.max(
+      hoursBetween(snapshotSettings.startDate, snapshotSettings.endDate),
+      1
+    ),
+    variations: snapshotSettings.variations.map((v, i) => ({
+      ...v,
+      name: variationNames[i] || v.id,
+    })),
+    analyses: analysisSettings,
```

Comment:
Changed this to just use the` ExperimentSnapshotAnalysisSettings` instead of building some ad-hoc type, and then have the default fallbacks in `analyzeExperimentMetric`, where we were doing them already anyways.